### DISCUSSION TITLE
Asking of how to save the predicted mask and image from registration model as Nifti image

### DISCUSSION THREAD

**Discussion by JosianeUwineza** [Project-MONAI/MONAI#6287] [Q&A] [answered]
Hi Team,  I want to thank for good library  with good feature. My Question is about  how can I save the predicted images and masks so that I can be able to check if registration  was successful performed using 3D slicer?  I trained model using LocalNed model and to get the pair data I rotated image  and mask by 20 degrees, by evaluating the model I got 0.97 mean dice score. My question is about how can I save the predicted image and masks?

Your help will be much appreciated!

**Comment by wyli** [comment, marked as answer]
please make sure the data shape: [channels, spatial_dim_1, spatial_dim_2, ...], then you can use
```py
saver = SaveImage(resample=False)
saver(data)
```

more examples in this tutorial https://github.com/Project-MONAI/tutorials/blob/main/modules/load_medical_images.ipynb

**Comment by JosianeUwineza** [comment]
Thanks @wyli, It works.

**Comment by JosianeUwineza** [comment]
I found that the affine of the predicted image after registration is the same as the affine of the moving image.  Is it normal that the monai registration prediction affine matrix to be the same as the moving affine matrix?  Due to that when I saw that the predicted image is same as moving image which means that the registration did not happen. 

Team help me to understand this
